nemo/collections/asr/models/ctc_bpe_models.py

```diff
@@ -67,22 +67,38 @@ def __init__(self, cfg: DictConfig, trainer=None):
         vocabulary = self.tokenizer.tokenizer.get_vocab()
 
         # Set the new vocabulary
-        cfg.decoder.params.vocabulary = ListConfig(list(vocabulary.values()))
+        if "params" in cfg.decoder:
+            cfg.decoder.params.vocabulary = ListConfig(list(vocabulary.values()))
+        else:
+            cfg.decoder.vocabulary = ListConfig(list(vocabulary.values()))
 
         # Override number of classes if placeholder provided
-        if cfg.decoder.params['num_classes'] < 1:
+        if "params" in cfg.decoder:
+            num_c = cfg.decoder["params"]["num_classes"]
+        else:
+            num_c = cfg.decoder["num_classes"]
+
+        if num_c < 1:
             logging.info(
                 "\nReplacing placeholder number of classes ({}) with actual number of classes - {}".format(
-                    cfg.decoder.params['num_classes'], len(vocabulary)
+                    num_c, len(vocabulary)
                 )
             )
-            cfg.decoder.params['num_classes'] = len(vocabulary)
+            if "params" in cfg.decoder:
+                cfg.decoder["params"]["num_classes"] = len(vocabulary)
+            else:
+                cfg.decoder['num_classes'] = len(vocabulary)
 
         super().__init__(cfg=cfg, trainer=trainer)
 
         # Setup metric objects
         self._wer = WERBPE(
-            tokenizer=self.tokenizer, batch_dim_index=0, use_cer=False, ctc_decode=True, dist_sync_on_step=True,
+            tokenizer=self.tokenizer,
+            batch_dim_index=0,
+            use_cer=False,
+            ctc_decode=True,
+            dist_sync_on_step=True,
+            log_prediction=self._cfg.get("log_prediction", False),
```

Comment:
Hmm if we are doing this, might as well go all the way and also do `use_cer=self._cfg.get('use_cer', False)`

Reply:
done